Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -34,11 +47,12 @@ public class TapToPlace : MonoBehaviour, IInputClickHandler
         private SpatialMappingManager spatialMappingManager;
 
         /// <summary>
-        /// Keeps track of if the user is moving the object or not.
+        /// Keeps track of which object we are placing. By default this is equal to
+        /// the current gameObject, but can be changed to a parent gameObject.
         /// </summary>
-        private bool placing;
+        private GameObject objectToPlace;
```

Comment:
I think it's meant to replace line 118 where you've pointed out we still only update the game object the script is attached to.